### ISSUE TITLE
plot still starts at n = 7 but should be updated number of factors.

### ISSUE THREAD

**Issue by Lakens** [Lakens/ANOVApower#20]

**Comment by Lakens** [comment]
Store Lambda_2 in power_oneway_within
Add alpha sig to ANOVA_exact power

**Comment by Lakens** [comment]
Should be solved by 
min_n = 7 replace by 
min_n = prod(as.numeric(unlist(regmatches(design_result$design,
                                       gregexpr("[[:digit:]]+", design_result$design)))))
right @arcaldwell49

**Comment by arcaldwell49** [comment]
No, the plot *should* still start at 7 the function can handle small sample sizes even when ANOVA_exact cannot.
